beeai/agents/rebase_agent.py

```diff
@@ -71,13 +72,18 @@ def get_instructions() -> str:
 
       6. Generate a SRPM using `centpkg --release <DIST_GIT_BRANCH> srpm`.
 
+      7. In your output, provide a "files_to_git_add" list containing all files that should be git added for this rebase.
+         This typically includes the updated .spec file and any new/modified/deleted patch files or other files you've changed
+         or added/removed during the rebase. Do not include files that were automatically generated or downloaded by spectool.
```

Comment:
I thought `git add` can't be used to add removed files to index, but apparently it can, with the [`--update`](https://git-scm.com/docs/git-add#Documentation/git-add.txt---update) flag.